lib/oc-tests/test_shareDir.py

```diff
@@ -95,6 +95,13 @@ def setup(step):
     check_users(config.oc_number_test_users)
 
     reset_rundir()
+    reset_server_log_file()
```

Comment:
Next I want to: 
1) config.oc_number_test_users : make this option None by default (you would need to enable mutliuser mode explictly in your configuration) 
2) separate some of your specific functions from __init__.py so it would be easier to work with pull requests